static/app/components/dropdownButtonV2.tsx

```diff
@@ -58,7 +58,8 @@ const StyledChevron = styled(IconChevron)`
   flex-shrink: 0;
 `;
 
-const StyledButton = styled(Button)<
+// this is exported so we can better control the button style.
+export const StyledButton = styled(Button)<
```

Comment:
Can we give this a better name in that case? Having `StyledButton` exported in the global namespace could be confusing.

How about `DropdownTriggerButton`?

Reply:
Done